EIPS/eip-7093.md

```diff
@@ -0,0 +1,409 @@
+---
+eip: 7093
+title: Social Recovery Interface
+description: Interfaces for social recovery account supporting various guardian types and customizable recovery policies.
+author: John Zhang (@johnz1019), Davis Xiang (@xcshuan), Kyle Xu (@kylexyxu), George Zhang (@odysseus0)
+discussions-to: https://ethereum-magicians.org/t/eip-social-recovery-interface/14494
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-05-29
+---
```

Comment:
If you recommend that implementations support EIP-1271 signatures, you should require that EIP here:

```suggestion
requires: 1271
---
```